.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: release
+
+on:
+  push:
```

Comment:
We'll want on release, not on tag. Don't want to trigger a release when we cut, e.g. a `v7` legacy tag when we start v8 development: https://github.com/nwithan8/pytulli/blob/master/.github/workflows/publish.yml#L7

Reply:
I think we should also have a manual workflow dispatch as well, just in case something goes wrong we can always manually intervene.

Reply:
Ah indeed, nice catch on the old branches.

Workflow dispatch added.